docs/advanced-topics.asciidoc

```diff
@@ -3,8 +3,35 @@
 
 * <<custom-transactions, Custom Transactions>>
 * <<custom-transaction-name, Initial page load transaction names>>
+* <<longtasks>>
 * <<typescript>>
```

Comment:
Kind of unrelated to this PR, but since we're touching this file, let's move the "custom-*" links to the bottom of the list and remove the custom titles so we match the order in the docs:
```
* <<longtasks>>
* <<typescript>>
* <<custom-transactions>>
* <<custom-transaction-name>>
```